weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableQuery.ts

```diff
@@ -130,6 +135,37 @@ export const useCallsForQuery = (
     }
   );
 
+  const storageSizeCols = useMemo(() => ['id', 'total_storage_size_bytes'], []);
+  const storageSizeFilter = costFilter;
+
+  const storageSize = useCalls(
```

Comment:
I would recommend trying to merging this storage second query with the cost second query. This would allow you to inly have a single second query

Reply:
I see the comment below indicates you did not want to do this (perhaps it was slower?)